notebooks/02_model/baseline_deep_dive.ipynb

```diff
@@ -196,7 +195,7 @@
     "    header=['UserId', 'MovieId', 'Rating', 'Timestamp']\n",
     ")\n",
     "\n",
-    "data.head()"
+    "display(data.head())"
```

Comment:
about `display`, removing it would give the exact same result as `data.head()`, is that right? if so, I think we should remove it for consistency with the rest of the notebooks

Reply:
I personally like to call 'display' explicitly for two reasons,
1. It shows my intention explicitly, like "I'm gonna display this data here"
2. If the cell printout some lines after data.head(), dataframe will be not showing. If use display(data.head()) and printout texts after that, both are shown up correctly.

But also agree would be good to have consistency. What do you think? I will follow Rome's rule ;-)

Reply:
I'm ok with whichever you prefer, but I would say we should have consistency, so all with display or all without display. Up to you :-)

Reply:
yes sir! I'm changing the notebook. will ping you once done!